### PR TITLE
Add quickstart guide

### PR DESCRIPTION
**Pull request by ferozjilla**
Quickstart that covers - 
- installing the cluster operator
- deploying a simple rabbitmq cluster using the operator
- accessing management ui on the deployed cluster
- viewing rabbitmq logs on the deployed cluster
- links to next steps

It also 
- breaks down the kubernetes resources that get created as a result of both installing the operator and creating a rabbitmq cluster using the operator
- promotes the examples on the cluster operator github repo

### CODE REVIEW

**Review by lukebakken — COMMENTED** [review]
I'll test this with `kind` today.

File: site/kubernetes/operator/quickstart-operator.md [hunk 1]
```diff
@@ -0,0 +1,237 @@
...
+
+```yaml
+apiVersion: rabbitmq.com/v1beta1
+kind: RabbitmqCluster
+metadata:
+	name: hello-world
+```
+Submit this using the following command -
+```shell
+kubectl apply -f https//raw.githubusercontent.com/rabbitmq/cluster-operator/main/docs/examples/hello-world/rabbitmq.yaml
```
Comment (lukebakken) [inline site/kubernetes/operator/quickstart-operator.md]: `https//` -> `https://`

File: site/kubernetes/operator/quickstart-operator.md [hunk 2]
```diff
@@ -0,0 +1,237 @@
...
+
+Next step, would be to connect an application to the RabbitMQ Cluster in order to use it's messaging capabilities. The [perf-test](https://github.com/rabbitmq/rabbitmq-perf-test) application is frequently used within the RabbitMQ community for load testing a RabbitMQ Cluster.
+
+Here, we will be using the `hello-world` service to find the connection address, and the `hello-world-default-user` to find connection credentials.
+
+```shell
+username=$(kubectl get secret hello-world-default-user -o jsonpath='{.data.username}' | base64 --decode)
+password=$(kubectl get secret hello-world-default-user -o jsonpath='{.data.password}' | base64 --decode)
+service=$(kubectl get service hello-world -o jsonpath='{.spec.clusterIP}')
+kubectl run perf-test --image=pivotalrabbitmq/perf-test -- --uri amqp://{username}:{password}@{service}
```
Comment (lukebakken) [inline site/kubernetes/operator/quickstart-operator.md:203]:
```
username="$(kubectl get secret hello-world-default-user -o jsonpath='{.data.username}' | base64 --decode)"
password="$(kubectl get secret hello-world-default-user -o jsonpath='{.data.password}' | base64 --decode)"
service="$(kubectl get service hello-world -o jsonpath='{.spec.clusterIP}')"
kubectl run perf-test --image=pivotalrabbitmq/perf-test -- --uri "amqp://$username:$password@$service"
```

**Review by lukebakken — COMMENTED** [review]
This is great and works as described using `kind` on Arch Linux. There are just a couple minor edits.

**Comment by ferozjilla** [comment]
Thanks for the quick review Luke! I'll make the minor edits and push again.

File: site/kubernetes/operator/quickstart-operator.md [hunk 3]
```diff
@@ -0,0 +1,237 @@
...
+
+You may also use the kubectl rabbitmq plugin to list the RabbitMQ Clusters deployed -
+
+```shell
+kubectl rabbitmq list
+NAME          AGE    STATUS
+hello-world   4m10s
+```
+
+Briefly breaking down what the `hello-world` RabbitMQ Cluster has created in terms of Kubernetes objects -
```
Comment (ChunyiLyu) [inline site/kubernetes/operator/quickstart-operator.md:127]: I wonder this section on child resources could be too much details for a `quick start`. I think we talk about child resources in other docs as well, so maybe we can skip on the information here to keep quick start simple and short.

**Review by ChunyiLyu — COMMENTED** [review]
Great document in general! I have a general suggestion/observation. For me a quick start guide should be easy to follow and concise. We have other documentations that go into more details about different use cases. Looking at the guide now:

1. Install the RabbitMQ Cluster Operator
2. Deploy a RabbitMQ Cluster using the Operator
3. View RabbitMQ Logs
4. Access the RabbitMQ Management UI
5. Attach A Workload to the Cluster
6. Next Steps

1 & 2 are necessary. But 3, 4, 5 are all about using the created rabbitmq. I wonder whether we can delete the log section or the management section from quick start, and move it to somewhere else (I am pretty sure that we talk about how to get logs and access management in other parts already)?

File: site/kubernetes/operator/quickstart-operator.md (hunk 3 above)
Comment (ferozjilla) [reply site/kubernetes/operator/quickstart-operator.md:127]: I read the document again and agree with you. I think it's better to have a dedicated document that focuses on this than to cover it briefly here. Will trim it.

**Comment by ferozjilla** [comment]
Hey @ChunyiLyu, thanks for reviewing :)

I looked in our current docs and we only have section 5 (connect using perf-test). Sections 3 (rabbitmq logs) and 4 (access management ui) are missing. They could both live in the Using docs: https://www.rabbitmq.com/kubernetes/operator/using-operator.html

Given that the sections (3,4,5) are so small, I am okay having them in the quickstart too, since it will keep the flow of the reader. However, we should also have them in the Using docs: https://www.rabbitmq.com/kubernetes/operator/using-operator.html

What do you think?

**Comment by ChunyiLyu** [comment]
@ferozjilla Happy to merge the doc as it is now. If we do decide the quick start is too verbose, it's easy to move/delete the steps :)

**Pull request merged**